src/components/ProjectMembersSummary/ProjectMembersSummary.tsx

```diff
@@ -15,6 +15,7 @@ export const ProjectMembersSummary: FC<ProjectMembersSummaryProps> = ({
     ({ user }): MemberSummaryItem => ({
       avatarLetters: user?.value?.avatarLetters ?? undefined,
       label: user?.value?.firstName ?? '',
+      id: user?.value?.id || '',
```

Comment:
The member list should probably be filtered to only the members that the current user has access to read.

Reply:
Added the filter